helm-charts/metal3-deploy/values.yaml

```diff
@@ -10,6 +10,26 @@
 
 # Define a global default for other charts to leverage.
 global:
+  # whether to enable external-dns
```

Comment:
I think this need more details: Something like:

"external-dns is required unless there are pre-existing methods the create DNS hostname records for the ironic ingress resources that are used in bare metal provisioning.  If using external-dns without pdns, a supported external-dns DNS integration must be configured. See for https://github.com/kubernetes-sigs/external-dns for additional details"

Reply:
also this needs to be updated if they are using their own  DNS servers.  We added this as by default it used the DNS server for the cluster/host and that may not be the correct server. 

  # If using pdnsIP, update the nameservers to the pdnsIP address
  dnsConfig:
    nameservers:
    - 10.43.255.254
    searches:
    - suse.baremetal

Reply:
Thanks for the suggestion. I'll add it to the comments.